webbpsf/trending.py

```diff
@@ -1897,7 +1897,7 @@ def show_wfs_during_program(
 
     # Look up wavefront sensing and mirror move corrections for that range
     opdtable = get_opdtable_for_daterange(start_date, end_date)
-    corrections_table = webbpsf.mast_wss.get_corrections(opdtable)
+    corrections_table = webbpsf.mast_wss.get_corrections(opdtable) # noqa TODO - this is unused and has no ramifications, is there a reason to keep or can I delete this line?
```

Comment:
Probably leftover from copy/paste when developing this function. I think OK to delete